CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_mailchimp_source v0.5.1
+
```

Comment:
Be sure to include this in a section. I would maybe classify this as a feature update as this isn't really a bug we are fixing. Just updating the test to be more accurate.

Reply:
Added under Feature!